hknweb/candidate/templates/candidate/401.html

```diff
@@ -0,0 +1,35 @@
+{% extends "base.html" %}
```

Comment:
Any reason we can't make this a top-level 401 page, not just a candidate 401 page?

Reply:
Not really, I think I just did this because it was easier